### ISSUE TITLE
Indicator placement in Unity

### ISSUE THREAD

**Issue by davidemars** [nitroshare/nitroshare-desktop#32]
Hi,

What a great tool: thank you to everyone who makes the magic happen! I do have a question. I am running Ubuntu 14.10 with Unity, and have my windows controls on the left. When running Nitroshare, the indicator sits to the far left, obscuring the close window ("x") button. Is this something unique to my machine or expected behavior? And either way, is there a way to relocate the indicator?

This certainly isn't of great priority. Thank you again!

David

**Comment by nathan-osman** [comment]
Thank you for the kind words :smile:

The problem you described sounds like an issue with the AppIndicator. NitroShare is designed to automatically use an AppIndicator when run under Unity, but it sounds like this didn't happen. Can you please confirm which version of NitroShare you have installed with the following command:

    dpkg -s nitroshare | grep -i version

**Comment by davidemars** [comment]
Thank you for such a quick reply! I am running version 0.3.0. I 
installed the current version using the binary package from the 
NitroShare web site, but also installed using a PPA 
(ppa:george-edison55/nitroshare). On that install the behavior was the 
same (but I do not know which version was installed when I used the PPA).

I don't know if there is much value in seeing the actual behavior, but I 
am attaching a screen shot just the same. If you need any other details 
then just let me know.

**Comment by nathan-osman** [comment]
Unfortunately, I'm not sure what would cause the application to use a system tray icon instead of an AppIndicator. There is a workaround you can use in the meantime to force NitroShare to display an AppIndicator. Press <kbd>Alt</kbd>+<kbd>F2</kbd> and enter `env XDG_CURRENT_DESKTOP=unity nitroshare`.

**Comment by davidemars** [comment]
No problem: the important thing is that the application works so well! 
I'll try to see if there may be something unique to my system causing 
the issue, too. If so, then I certainly will let everyone know. Thank 
you, again!

**Comment by jonnius** [comment]
Could you make sure that you have installed the package libappindicator1? I have seem a similar issue and the solution was to install this package.

**Comment by davidemars** [comment]
I have to apologize, as I updated to Ubuntu 15.04 this week by doing a clean installation. I downloaded the 0.3.0 DEB file and installed NitroShare without any problems; it's working exactly as expected. The libappindicator1 package was installed before adding NitroShare, so I cannot reproduce the behavior. 

Thank you very much for following up on this issue so thoroughly! I'm going to look to see if I don't have an image of a 14.04 VM to use in trying to reproduce what was seen initially. If I do, then I will let you know the results. 


----- Original Message -----
